robolectric-shadows/shadows-core/src/main/resources/org/robolectric/shadows/ShadowAccessibilityNodeInfo.java.vm

```diff
@@ -265,108 +342,226 @@ public class ShadowAccessibilityNodeInfo {
+  }
+
+  @Implementation
+  public void setEnabled(boolean enabled) {
+    propertyFlags = (propertyFlags & ~ENABLED_MASK) |
+        (enabled ? ENABLED_MASK : 0);
+  }
+
+  @Implementation
+  public boolean isEnabled() {
+    return ((propertyFlags & ENABLED_MASK) != 0);
+  }
+
+  @Implementation
+  public void setPassword(boolean password) {
+    propertyFlags = (propertyFlags & ~PASSWORD_MASK) |
+        (password ? PASSWORD_MASK : 0);
+  }
+
+  @Implementation
+  public boolean isPassword() {
+    return ((propertyFlags & PASSWORD_MASK) != 0);
+  }
+
+  @Implementation
+  public void setSelected(boolean selected) {
+    propertyFlags = (propertyFlags & ~SELECTED_MASK) |
+        (selected ? SELECTED_MASK : 0);
+  }
+
+  @Implementation
+  public boolean isSelected() {
+    return ((propertyFlags & SELECTED_MASK) != 0);
+  }
+
+  @Implementation
+  public void setAccessibilityFocused(boolean focused) {
+    propertyFlags = (propertyFlags & ~A11YFOCUSED_MASK) |
+        (focused ? A11YFOCUSED_MASK : 0);
+  }
+
+  @Implementation
+  public boolean isAccessibilityFocused() {
+    return ((propertyFlags & A11YFOCUSED_MASK) != 0);
+  }
+#if ($api >= 21)
+  @Implementation
+  public void setMultiLine(boolean multiLine) {
+    propertyFlags = (propertyFlags & ~MULTILINE_MASK) |
+        (multiLine ? MULTILINE_MASK : 0);
+  }
+
+  @Implementation
+  public boolean isMultiLine() {
+    return ((propertyFlags & MULTILINE_MASK) != 0);
+  }
+
+  @Implementation
+  public void setContentInvalid(boolean contentInvalid) {
+    propertyFlags = (propertyFlags & ~CONTENT_INVALID_MASK) |
+        (contentInvalid ? CONTENT_INVALID_MASK : 0);
+  }
+
+  @Implementation
+  public boolean isContentInvalid() {
+    return ((propertyFlags & CONTENT_INVALID_MASK) != 0);
+  }
+
+  @Implementation
+  public void setDismissable(boolean dismissable) {
+    propertyFlags = (propertyFlags & ~DISMISSABLE_MASK) |
+        (dismissable ? DISMISSABLE_MASK : 0);
+  }
+
+  @Implementation
+  public boolean isDismissable() {
+    return ((propertyFlags & DISMISSABLE_MASK) != 0);
+  }
+
+  @Implementation
+  public void setCanOpenPopup(boolean opensPopup) {
+    propertyFlags = (propertyFlags & ~CAN_OPEN_POPUP_MASK) |
+        (opensPopup ? CAN_OPEN_POPUP_MASK : 0);
+  }
+
+  @Implementation
+  public boolean canOpenPopup() {
+    return ((propertyFlags & CAN_OPEN_POPUP_MASK) != 0);
+  }
+#end
   public void setA11yFocusable(boolean focusable) {
-    actionFlags = (actionFlags & ~A11YFOCUSABLE_MASK) | 
+    propertyFlags = (propertyFlags & ~A11YFOCUSABLE_MASK) |
         (focusable ? A11YFOCUSABLE_MASK : 0);
   }
 
   public void setTextSelectionSetable(boolean isTextSelectionSetable) {
-    actionFlags = (actionFlags & ~TEXT_SELECTION_SETABLE_MASK) | 
+    propertyFlags = (propertyFlags & ~TEXT_SELECTION_SETABLE_MASK) |
         (isTextSelectionSetable ? TEXT_SELECTION_SETABLE_MASK : 0);
   }
+  
+  public void setOnPerformActionListener(OnPerformActionListener listener) {
```

Comment:
Can we move this somewhere more discoverable in the file?  In the middle of property setters/getters might be difficult to find.